lore/settings.py

```diff
@@ -104,7 +106,7 @@ def get_var(name, default):
     'guardian.backends.ObjectPermissionBackend',
 )
 
-ROOT_URLCONF = 'ui.urls'
+ROOT_URLCONF = 'lore.urls'
```

Comment:
I do not understand this division: why admin or as should be in lore.url? Plus I guess there was a reason why the urls have been moved to ui